code/ecdsap256/Hacl.P256.fsti

```diff
@@ -1,501 +1,345 @@
 module Hacl.P256
```

Comment:
This file combines the ECDSA and ECDH APIs in a single file.
Does this make sense in the long-run? 
I would expect ECDSA to be generic.

Reply:
it would be great to add a generic spec for ECDSA and ECDH, let's do in the next round. will file an issue for this